core/src/main/java/psiprobe/controllers/RenderChartController.java

```diff
@@ -162,9 +162,9 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request,
 
       final XYLine3DRenderer renderer = new XYLine3DRenderer();
       renderer.setDrawOutlines(true);
-      renderer.setLinesVisible(true);
-      renderer.setShapesVisible(true);
-      renderer.setStroke(new BasicStroke(2));
+      renderer.setSeriesLinesVisible(0, true);
+      renderer.setSeriesShapesVisible(0, true);
+      renderer.setSeriesStroke(0, new BasicStroke(2));
```

Comment:
Thanks for the help.  Ill give the for loop a try.

Get Outlook for Android<https://aka.ms/ghei36>



On Sun, Jul 17, 2016 at 2:28 PM -0400, "Mark Lewis" <notifications@github.com<mailto:notifications@github.com>> wrote:


In core/src/main/java/psiprobe/controllers/RenderChartController.java<https://github.com/psi-probe/psi-probe/pull/746#discussion_r71086971>:

> @@ -162,9 +162,9 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request,
>
>        final XYLine3DRenderer renderer = new XYLine3DRenderer();
>        renderer.setDrawOutlines(true);
> -      renderer.setLinesVisible(true);
> -      renderer.setShapesVisible(true);
> -      renderer.setStroke(new BasicStroke(2));
> +      renderer.setSeriesLinesVisible(0, true);
> +      renderer.setSeriesShapesVisible(0, true);
> +      renderer.setSeriesStroke(0, new BasicStroke(2));


Just looking at the API, it kinda looks like these should be done in a for-loop now:

for (int i = 0; i < seriesMaxCount; i++) {
  renderer.setSeriesLinesVisible(i, true);
  renderer.setSeriesShapesVisible(i, true);
  renderer.setSeriesStroke(i, new BasicStroke(2));
}

I'd also suggest testing it locally just to be sure it still works.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/psi-probe/psi-probe/pull/746/files/5ddd1213728aa2ebc18438f1f20d7d6fce212567#r71086971>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AA7howBzD1vV3yzj36yE_kbQoieoSxaIks5qWnQ3gaJpZM4JOHYM>.